src/modules/displayList.js

```diff
@@ -18,6 +18,12 @@ const createTodo = () => {
       const checkbox = document.createElement('input');
       checkbox.type = 'checkbox';
       checkbox.id = 'checkbox';
+      if (a.completed === true) { checkbox.checked = 'checked'; }
+
+      checkbox.onclick = (e) => {
+        todo.completedTodo(e.target.checked, a.index);
+      };
```

Comment:
Your onlick event on checkboxes is triggering properly and working as expected ✔